tethys_cli/scaffold_templates/app_templates/reactpy/setup.py_tmpl

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_namespace_packages
```

Comment:
I think we're going to need to move to `pyproject.toml` sooner that later. Should we just start this scaffold out that way so we don't have to migrate it later?

Reply:
That makes sense to me. I don't know much about pyproject.toml, but my guess is it'd be a quick learn, yeah? I'll make it happen.